frontend/javascripts/oxalis/model/sagas/proofread_saga.ts

```diff
@@ -268,22 +265,18 @@ function* splitOrMergeAgglomerate(action: MergeTreesAction | DeleteEdgeAction) {
   const { tracingId: volumeTracingId } = volumeTracing;
 
   const layerName = volumeTracingId;
-  const activeMappingByLayer = yield* select(
-    (state) => state.temporaryConfiguration.activeMappingByLayer,
-  );
-  const mappingInfo = getMappingInfo(activeMappingByLayer, layerName);
-  const { mappingName, mappingType, mappingStatus } = mappingInfo;
-  if (
-    mappingName == null ||
-    mappingType !== "HDF5" ||
-    mappingStatus === MappingStatusEnum.DISABLED
-  ) {
-    Toast.error("An HDF5 mapping needs to be enabled to use the proofreading tool.");
-    return;
-  }
+  const isHdf5MappingEnabled = yield* call(ensureHdf5MappingIsEnabled, layerName);
+  if (!isHdf5MappingEnabled) return;
 
   if (!volumeTracing.mappingIsEditable) {
-    yield* call(createEditableMapping);
+    try {
+      yield* call(createEditableMapping);
+    } catch (e) {
+      console.error(e);
+      // Undo the last splitting/merging action since the proofreading action did not succeed
```

Comment:
what would be an example reason for this to fail?

Reply:
If the user brushed before, the creation of the editable mapping will fail. That's the main case.